POSTER_PROJECTS/README.md

```diff
@@ -53,53 +53,38 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
 - Take a flip chart, markers, etc and work wherever you want to work.
 
 ## Topics
-
-| # | Topic | Description |
+| # | Topic | Owner |
```

Comment:
@djdv Can you add the owners to each topic soon? Chris needs that info for the website

Reply:
Status will be posted here: https://gist.github.com/djdv/f8712c70211848e7af23cd8d86c2b320
and committed to the branch when confirmed.